### PR TITLE
Playback Speed

### PR DESCRIPTION
**Pull request by Arseny271**

### CODE REVIEW

File: app/src/main/java/org/thunderdog/challegram/navigation/MenuMoreWrapAbstract.java
Comment (vkryl) [inline app/src/main/java/org/thunderdog/challegram/navigation/MenuMoreWrapAbstract.java:1]: Missing copyright comment.

File: app/src/main/java/org/thunderdog/challegram/navigation/PlaybackSpeedLayout.java
Comment (vkryl) [inline app/src/main/java/org/thunderdog/challegram/navigation/PlaybackSpeedLayout.java:1]: Missing copyright comment.

File: app/src/main/java/org/thunderdog/challegram/util/CounterPlaybackSpeedDrawableSet.java
Comment (vkryl) [inline app/src/main/java/org/thunderdog/challegram/util/CounterPlaybackSpeedDrawableSet.java:1]: Missing copyright comment.

File: app/src/main/java/org/thunderdog/challegram/util/text/counter/CounterDrawableSet.java
Comment (vkryl) [inline app/src/main/java/org/thunderdog/challegram/util/text/counter/CounterDrawableSet.java:1]: Missing copyright comment.

File: app/src/main/java/org/thunderdog/challegram/util/text/counter/CounterTextPart.java
Comment (vkryl) [inline app/src/main/java/org/thunderdog/challegram/util/text/counter/CounterTextPart.java:1]: Missing copyright comment.

**Review by vkryl — APPROVED** [review]

**Review by vkryl — APPROVED** [review]

**Comment by cumcum8197** [comment]
223c39b495b6e2eadd8af00c9b837eb02da99e90

**Pull request merged**